src/components/header/components/shared/icons/SignUpIcon.component.tsx

```diff
@@ -0,0 +1,13 @@
+import type { CustomClasses } from '@/interfaces/CustomClasses.ts';
+
+export const SignUpIconComponent = ({ customClasses }: CustomClasses) => (
+    <svg className={customClasses} width="21" height="17" viewBox="0 0 21 17" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
You can export svg files from a figma-like file. And use with import. You can check how use others students.